assets/src/edit-story/components/canvas/pagenav/index.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app';
+
+const Wrapper = styled.div`
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+	height: 100%;
+	color:  ${ ( { theme } ) => theme.colors.fg.v1 };
+`;
+
+const Space = styled.div`
+	flex: 0 1 ${ ( { direction } ) => 'next' === direction ? 60 : 115 }px;
+`;
+
+const IconStyle = css`
+	border: solid ${ ( { theme } ) => theme.colors.fg.v1 };
+	border-width: 0 1px 1px 0;
+	display: inline-block;
+	padding: 3px;
+	cursor: pointer;
+	opacity: .25;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	color: inherit;
+	height: 25px;
+    width: 25px;
+	&:hover {
+		color: inherit;
+		opacity: 1;
+	}
+`;
+
+const IconNext = styled.i`
```

Comment:
If it's clickable, it should be a `button`. And either have (hidden) textual content or an `aria-label` or `title`.

Reply:
Thanks for noting that. Updated to using `a` instead, for convenience reasons. Q: Is there a reason why a button would be better?

Reply:
_Changing to button_